### PR TITLE
test arc

### PR DESCRIPTION
**Pull request by yujyuj**
added testing logic for test_arc

### CODE REVIEW

**Comment by novialriptide** [comment]
It seems as if the `todo_test_arc()` function was created as a placeholder during its development and was forgotten to be removed.

**Comment by MyreMylar** [comment]
> It seems as if the `todo_test_arc()` function was created as a placeholder during its development and was forgotten to be removed.

All the `todo_` prefixed tests were originally created as reminders to make a good test for this later, however in the case of arc, I agree that this function already looks to be well tested by the existing entire class of tests.

I don't think this test is adding anything new to the arc testing, and it's also saving a file for no obvious reason.

This PR could be repurposed to jut delete the redundant `todo_test_` chunk instead if you like @yujyuj

File: test/draw_test.py [hunk 1]
```diff
@@ -6186,9 +6186,42 @@ def test_arc__invalid_color_formats(self):
             with self.assertRaises(TypeError):
                 bounds_rect = self.draw_arc(**kwargs)
 
-    def todo_test_arc(self):
+    def test_arc(self):
         """Ensure draw arc works correctly."""
-        self.fail()
```
Comment (MyreMylar) [inline test/draw_test.py:6191]: Instead, just remove:

```
def todo_test_arc(self):
     self.fail()
```

**Review by MyreMylar — CHANGES_REQUESTED** [review]
I think just removing the `todo_test_arc(self)` statement entirely is the way to go here. `draw.arc()` has a large number of tests in the class this test is being added to.

File: test/draw_test.py (hunk 1 above)
Comment (Starbuck5) [reply test/draw_test.py:6191]: Isn't that what they did?

File: test/draw_test.py [hunk 2]
```diff
@@ -6186,9 +6186,42 @@ def test_arc__invalid_color_formats(self):
...
+        surface.fill(black)
+
+        # rectangle that contains for the ellipse arc.
+        # 0 pixel from left, 0 pixel from top
+        # 80 pixels wide, 40 pixels high
+        rect = (0, 0, 80, 40)
+
+        # angle of the arc in radians
+        start_angle = 0.0
+        stop_angle = 3.14
```
Comment (novialriptide) [inline test/draw_test.py:6206]: There should be a builtin function that defines pi.

File: test/draw_test.py (hunk 1 above)
Comment (MyreMylar) [reply test/draw_test.py:6191]: they also added a test here - but draw arc is already extensively tested.

**Pull request closed without merging**